### PR TITLE
add ivy api - loss function: NLL function

### PR DESCRIPTION
**Pull request by MarkCodering**
# PR Description

## Related Issue

Close nll_loss #26776

## Checklist

- [x] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

**Comment by MarkCodering** [comment]
@MuhammedAshraf2020 Could you help me review the PR? Thanks

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi @MarkCodering 
Thanks for the PR. Not sure why you did not get a review. Apologies from our end.
Few requests. 
Could we not change the docstring for cross entropy with values "input" being changed to "iivyut" for some reason. Could we also include the test for any new function you added like nll loss. Thank you very much :)

**Comment by Ishticode** [comment]
Closing this due to inactivity. Please feel free to reopen if you would like to continue working on this. Thanks :)
